templates/tensorflow-gpu-1.12/src/__ALGO___test.py

```diff
@@ -0,0 +1,15 @@
+from .__ALGO__ import apply
+
+"""
+This is the testing file, which should be bundled along with the Algorithm.py file when creating a template.
+These tests are run during the creation of a template, and if they fail - the templating process fails as well.
+"""
+
+def primariy_api_example():
+    input = {'image_url': 'https://s3.amazonaws.com/algorithmia-uploads/money_cat.jpg'}
```

Comment:
would be nice if we could put this behind a DNS entry in the future. Not a blocker for this PR tho

Reply:
I agree 100%

Reply:
Created https://algorithmia.atlassian.net/browse/ALGO-531